infra/aws/e2e/accounts/attach.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+set -x
+
+for s in a b c d; do
+  ID=e2e-kops-2020-03-15-$s
+
+  for r in AmazonEC2FullAccess IAMFullAccess AWSDeepRacerCloudFormationAccessPolicy; do
```

Comment:
Not directly related to this PR, but if we want to also use accounts created by this script for cluster-api-provider-aws, it would be nice to also add `AmazonSSMFullAccess` here, or a custom policy that would at least allow creation of Session Manager-initiated remote sessions.